kolibri/content/api.py

```diff
@@ -4,221 +4,173 @@
     """
-    Get a set of ContentMetadatas that have this ContentMetadata as the immediate parent.
+    Get a set of ContentNodes that have this ContentNode as the immediate parent.
 
     :param channel_id: str
-    :param content: ContentMetadata or str
-    :return: QuerySet of ContentMetadata
+    :param content: ContentNode or str
+    :return: QuerySet of ContentNode
     """
     return content.get_children().using(channel_id)
 
 @can_get_content_with_id
 def leaves(channel_id=None, content=None, **kwargs):
     """
-    Get all ContentMetadatas that are the terminal nodes and also the descendants of the this ContentMetadata.
+    Get all ContentNodes that are the terminal nodes and also the descendants of the this ContentNode.
 
     :param channel_id: str
-    :param content: ContentMetadata or str
-    :return: QuerySet of ContentMetadata
+    :param content: ContentNode or str
+    :return: QuerySet of ContentNode
     """
     return content.get_leafnodes().using(channel_id)
 
 @can_get_content_with_id
-def get_all_formats(channel_id=None, content=None, **kwargs):
-    """
-    Get all possible formats for a particular content including its descendants' formats.
-
-    :param channel_id: str
-    :param content: ContentMetadata or str
-    :return: QuerySet of Format
-    """
-    all_end_nodes = leaves(channel_id=channel_id, content=content)
-    return KolibriContent.Format.objects.using(channel_id).filter(contentmetadata__in=all_end_nodes)
-
-@can_get_content_with_id
-def get_available_formats(channel_id=None, content=None, **kwargs):
-    """
-    Get all available formats for a particular content excluding its descendants' formats.
-    if the pass-in content is a topic, this function will return null.
-
-    :param channel_id: str
-    :param content: ContentMetadata or str
-    :return: QuerySet of Format
-    """
-    return KolibriContent.Format.objects.using(channel_id).filter(contentmetadata=content, available=True)
-
-@can_get_content_with_id
-def get_possible_formats(channel_id=None, content=None, **kwargs):
-    """
-    Get all possible formats for a particular content excluding its descendants' formats.
-    if the pass-in content is a topic, this function will return null.
-
-    :param channel_id: str
-    :param content: ContentMetadata or str
-    :return: QuerySet of Format
-    """
-    return KolibriContent.Format.objects.using(channel_id).filter(contentmetadata=content)
-
-@can_get_content_with_id
-def get_files_for_quality(channel_id=None, content=None, format_quality=None, **kwargs):
+def get_files_for_preset(content=None, preset=None, **kwargs):
     """
     Get all files for a particular content in particular quality.
     For format_quality argument, please pass in a string like "high" or "low" or "normal".
     topic content will return null.
 
     :param channel_id: str
-    :param content: ContentMetadata or str
+    :param content: ContentNode or str
     :param format_quality: str
```

Comment:
Doesn't seem like we use `format_quality` anymore. Let's remove mentions of that in the docstring then.